src/apps/testapps/testVertex.c

```diff
@@ -63,4 +63,28 @@ SUITE(Vertex) {
             vertexNumForDirection(pentagon, K_AXES_DIGIT) == INVALID_VERTEX_NUM,
             "K direction on pentagon should return invalid vertex");
     }
-}
\ No newline at end of file
+
+    TEST(directionForVertexNum_hex) {
+        H3Index origin = 0x823d6ffffffffff;
+        Direction dirs[NUM_DIGITS] = {0};
```

Comment:
Looks like the type of `dirs` is actually boolean, it's instead indexed by `Direction`
Nit: `dirsSeen`?

Reply:
Good call, will fix.